bucket/scoop-search.json

```
@@ -0,0 +1,13 @@
+{
+    "version": "0.6.5",
+    "license": "MIT",
+    "url": "https://github.com/tokiedokie/scoop-search/releases/download/v0.6.5/scoop-search.exe",
+    "homepage": "https://github.com/tokiedokie/scoop-search",
+    "description": "yet another scoop search",
```

```suggestion
    "version": "0.6.5",
    "description": "Yet another scoop search",
    "homepage": "https://github.com/tokiedokie/scoop-search",
    "license": "MIT",
    "url": "https://github.com/tokiedokie/scoop-search/releases/download/v0.6.5/scoop-search.exe",
```

version -> description -> homepage -> license ...